Jenkinsfile

```diff
@@ -46,132 +49,107 @@ pipeline {
       steps {
         script {
           // Build site with CV Buildkit
-          sh "civibuild create ${params.CIVIHR_BUILDNAME} --type hr17 --civi-ver 4.7.27 --hr-ver staging --url $WEBURL --admin-pass $ADMIN_PASS"
+          sh "civibuild create ${params.CIVIHR_BUILDNAME} --type drupal-clean --civi-ver 4.7.27 --url $WEBURL --admin-pass $ADMIN_PASS"
+
           sh """
             cd $DRUPAL_MODULES_ROOT/civicrm
             wget -O attachments.patch https://gist.githubusercontent.com/davialexandre/199b3ebb2c69f43c07dde0f51fb02c8b/raw/0f11edad8049c6edddd7f865c801ecba5fa4c052/attachments-4.7.27.patch
             patch -p1 -i attachments.patch
             rm attachments.patch
           """
 
-          // Change git remote of civihr ext to support dev version of Jenkins pipeline
-          changeCivihrGitRemote()
-
-          // Get repos & branch name
+          // Get target and PR branches name
```

Comment:
Why was `changeCivihrGitRemote` removed?

Reply:
Because it would not work as expected. That function existed as an attempt to support pull requests from forks, but it would not really work as it should. In cases like that, the process is a bit more complex (we need to merge code from two different remotes) and, since it's out of the scope of this PR, I decided to remove the code altogether. I'll add note of this to the PR description

Reply:
Thanks, the description was helpful